### ISSUE TITLE
DCT: Faulty Advice from Output File... Any Good Replacements?

### ISSUE THREAD

**Issue by JonathonMisiewicz** [psi4/psi4#1354]
I'm attempting to converge a very tricky molecule with ODC-12. _Something_ goes wrong in the computation, likely involving the fact that I added damping, and I get the following error message:
```
                There was a problem correcting the MO phases.
                If this does not converge, try ALGORITHM=TWOSTEP
```

However helpful a change of algorithm might be, the two step algorithm isn't implemented for the orbital optimized methods, so this advice is useless for me, and aggravating for any users encountering this problem. So the error message needs to be changed if the cumulant parameterization includes orbital rotations. But it would be nice if I could replace it with something else. Do any of the (very few) people familiar with wrangling with tricky molecules in DCT have any suggestions?

**Comment by ssh2** [comment]
Good catch. Although there could be multiple suggestions, one that I found the most effective is to choose a different set of starting (guess) orbitals. This can be done by either setting odc_guess to true or by feeding in other (non-HF) orbitals. Either one is fine with me. If you choose to go with the first option, the second line can be modified as "If this does not converge, try ODC_GUESS=TRUE" (this obviously only makes sense for orbital-optimized methods).
